### PR TITLE
Add mattermost plugin todo v0.2.0

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR will add TODO to the marketplace as version `v0.2.0`

~~Note the additional change to `cmd/generate/main.go` which adds the new plugin.~~

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -3785,6 +3785,68 @@
...
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/tag/v0.2.0",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl6fK3MACgkQ0bVLR6XO/sR2JQ/8CUdVcqBLXStJE8g+VbHUUT0hfBnijIPzbiPVzHjnkWIz6kY6nUrfykg6nSNN5rhverw0ovOLEwrusrKKb0997xvn60Ekmsyfrr3b2V+ZCVvgdG678lJWXnhqu6n/xtCW6GCTTVZB42Jnr4Ri5VVwNL0mSE6Rc4urlWnMApasKVi4tl3trbvVyOJrKUwtlgNXHYnRkZOVcmc64AxG7DBhXawmuNTUbZ6f9WaC8/F9ZxxErWxERN0rlH/rSdYsRG/CIIYxCJ63R9YT2xRMcJkiMBt2bh4zXKazHfw5/ozadyG7TEQVnPrNS+xZvj4JkgKLw5c3NwcIORONujnL8JINlOKUJf+xmBdNPGTwS2WWeDp6rSx0xPmSttXi8b+glBXvq80lrzSz1CLvSr/MOwB3yeuMl1EAKhPvwrUm49uGt2dsIgaHaImbAXXzs277OjuG4ra6bNKPpgjh7mjr5opnDRai3r1cqOMKPQnevBXxQCcRPbIUfxr3tVKOQSaG2cTFgot57yebnUGwYNFRD+D6oAkgTh2nIm95QPqLsQ3CpawPiR9pz9VIX0RVoQg0+OYKAGql8GjXH97cj4EZZnlDLtBIxol0mmGVxndNFYhYesnQxF9l2LroG/8jown+gqQorOcS5cv4RSGVl31QftWNgiDALr7mH2ionJFFn0vVjiA=",
+    "manifest": {
+      "id": "com.mattermost.plugin-todo",
+      "name": "Todo Plugin",
+      "description": "This plugin makes it easy to keep track of Todo issues and get daily reminders.",
+      "version": "0.2.0",
```
Comment (hanzei) [inline plugins.json:3798]: I'm wondering if we should make clear that this plugin is still in beta, breaking changes might happen and the plugin is not production ready. See also https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins


Please note that some code changes would be needed to add a "Beta" label. I'm happy to work on them if we decide to go this path.

Comment (hanzei) [reply plugins.json:3798]: Moving the discussion to https://community-release.mattermost.com/core/pl/mgrez65gjfngbbjqqho9ka1xte

**Comment by hanzei** [comment]
Added the label via `go run ./cmd/generator/ add mattermost-plugin-todo v0.2.0 --beta --officia`

**Comment by hanzei** [comment]
This should not be merged until https://mattermost.com/pl/default-beta-plugins is set up.

**Comment by hanzei** [comment]
@levb @larkox @aaronrothschild @DHaussermann https://mattermost.com/pl/default-beta-plugins is now set up. Please review as soon as you can so we can get this merged.

**Comment by DHaussermann** [comment]
@larkox there seems to be an issue that was missed earlier regarding enabling the plugin. Active users will see a JS error when the plugin is enabled by the admin and need to refresh to see the plugin in the UI. 

This may be an issue that occurs on other plugins as well. Can we discuss if this is solvable before we proceed with the market place deploy. 
EnableToDo

**Comment by jfrerich** [comment]
Thanks @DHaussermann!  @aaronrothschild, you have the final signoff and we'll market todo 2.0!

**Comment by hanzei** [comment]
I synced up offline with @aaronrothschild and he is fine with merging this.

**Pull request merged**